sixaxis.cpp

```diff
@@ -432,27 +428,20 @@ void do_rumble(int csk, int led_n, int weak, int strong, int timeout)
 
     setrumble[11] = ledpattern[led_n]; //keep old led
     send(csk, setrumble, sizeof(setrumble), 0);
-#ifndef GASIA_GAMEPAD_HACKS
-    unsigned char buf[128];
-    recv(csk, buf, sizeof(buf), 0); //MSG_DONTWAIT?
-#endif
+    if (!gasia) {
+        unsigned char buf[128];
+        recv(csk, buf, sizeof(buf), 0); //MSG_DONTWAIT?
```

Comment:
The recv command may break shanwan. It may also be safe to simply remove for all controllers, but I need to test that. If it breaks shanwan, then detection for shanwan can also be added here.